File: pos-ui/src/main/scala/ar/com/pos/db/SaleHbm.scala
package ar.com.pos.db

import java.util
import java.util.Date
import java.util.HashSet

class SaleHbm {
	
  var _idSale: Long = 0
  var _description: String = _
  var _date: Date = _
  var _detail: java.util.Set[SaleDetailHbm] = _
  var _totalAmount: Double = 0

  def getIdSale = _idSale
  def setIdSale (value:Long):Unit = {_idSale = value}

  def getDescription = _description
  def setDescription (value:String):Unit = {_description = value}
  
  def getDate = _date 
  def setDate (value:Date):Unit = {_date = value} 

  def getDetail = _detail
  def setDetail (value: java.util.Set[SaleDetailHbm]):Unit = {_detail  = value}

  def getTotalAmount = _totalAmount 
  def setTotalAmount (value:Double):Unit = {_totalAmount = value}
}

File: pos-backend/src/main/java/ar/com/terminal/db/SaleHbm.java
package ar.com.terminal.db;

import java.util.Date;
import java.util.HashSet;
import java.util.Set;

public class SaleHbm {

	private Long idsale;
	private Long idproduct;
	private String description;
	private Date date;
	private Set<ProductHbm> products = new HashSet<ProductHbm>();
	private Float totalAmount;
	
	public Set<ProductHbm> getProducts() {
		return products;
	}
	public void setProducts(Set<ProductHbm> products) {
		this.products = products;
	}
	public Long getIdsale() {
		return idsale;
	}
	public void setIdsale(Long idsale) {
		this.idsale = idsale;
	}
	public long getIdproduct() {
		return idproduct;
	}
	public void setIdproduct(long idproduct) {
		this.idproduct = idproduct;
	}
	public String getDescription() {
		return description;
	}
	public void setDescription(String description) {
		this.description = description;
	}
	public Date getDate() {
		return date;
	}
	public void setDate(Date date) {
		this.date = date;
	}
	public Float getTotalAmount() {
		return totalAmount;
	}
	public void setTotalAmount(Float totalAmount) {
		this.totalAmount = totalAmount;
	}
}
